kahuna/public/js/services/api/collections-api.js

```diff
@@ -42,12 +42,20 @@ apiServices.factory('collections', ['mediaApi', function (mediaApi) {
         });
     }
 
+    function addImagesToCollection(imageIds, collectionPath) {
+        const promises = imageIds.map(id => mediaApi.find(id).then(
+                image => image.perform('add-collection', {body: {data: collectionPath}})));
```

Comment:
Happy to go with that, maybe we could put a note in saying it's inefficient (talks to API when we have all the information) and why we can't do the right thing yet?